### ISSUE TITLE
Home page What's new <li> copy size too small

### ISSUE THREAD

**Issue by jeperry** [patternfly/patternfly-org#190]
Bullet items on 'What's new" should be brought up in font size to match <p> size elsewhere on the page. For example, match the text under "About PatternFly"

At the current (base default) size the bullet copy is overwhelmingly tiny compared to the rest of the content on the page, making it look unimportant.

**Comment by LHinson** [comment]
@amarie401 Given that we now have the design status page, I don't think we should keep this "What's New" section as it's just another place to maintain content. Can we look at removing this and just featuring the Featured Content?

**Comment by dabeng** [comment]
Hi @LHinson If we decide to remove "What's New" section, it will be a big change for [home page](http://www.patternfly.org/). I think it's better to create new layout of home page firstly and I'll send PR following the new design.

**Comment by deernano** [comment]
![Uploading homepage-featuredcontent@2x.png…]()
@LHinson @dabeng Here is a proposed 'What's new' section. The 'What's new' has been removed. The 'Featured content' turns to full screen. Changed the 'Lean more' link position and the carousel. Any comments?

**Comment by mcarrano** [comment]
I think this is fine @deernano . The only thing I wonder about is whether the carousel buttons should either have some drop shadow or if the screen shot should be made smaller so the buttons can go to either side. My concern is that the previous and next buttons might look like part of the screen to a user seeing this for the first time.

**Comment by deernano** [comment]
@mcarrano yes, you are right. How about change a color and add shadow for the carousel button as below?
![homepage-featuredcontent1 2x](https://user-images.githubusercontent.com/5396309/40343821-67bed340-5dc4-11e8-8b5b-2956af8ad22f.png)

**Comment by mcarrano** [comment]
Yes, I think that is better @deernano . These buttons probably also need a disabled state for when we are showing the first or last item.

**Comment by deernano** [comment]
@mcarrano @dabeng 
Disabled state for showing the first item
![homepage-featuredcontent2 2x](https://user-images.githubusercontent.com/5396309/40400478-e357acf6-5e74-11e8-9fb7-66a6beaac870.png)

Disabled state for showing the last item
![homepage-featuredcontent3 2x](https://user-images.githubusercontent.com/5396309/40400613-8d5f68d8-5e75-11e8-9803-d6f326e232da.png)

**Comment by deernano** [comment]
@mcarrano 
After discussing with @dabeng , I did another layout. Since the full screen casousel may feel too large on big screens, and the loading speed may become slower if we use more clear images,  I think we can  place the title / decription on the left side of the carousel as the below image shows.
Also, we might need to make sure the decription of each image is correct and accurate. The quickest way may be grab the first sentence in Overview. What do you think?

![homepage-featuredcontent1 2x](https://user-images.githubusercontent.com/5396309/40406896-077a2c60-5e95-11e8-9b10-8bf0c3faa249.png)
![homepage-featuredcontent2 2x](https://user-images.githubusercontent.com/5396309/40406902-09a980b2-5e95-11e8-8fc5-370081500110.png)

**Comment by mcarrano** [comment]
@deernano @dabeng I like this second proposal a lot better. Let's go forward with that. As for the text, agree that the best thing will be to pull this from the Overview page. How will you know how much text to include? Would we look at the first "n" characters or have some other method?

**Comment by jeperry** [comment]
I agree, very nicely done.

**Comment by dabeng** [comment]
@deernano @mcarrano Please review the [Demo](https://rawgit.com/dabeng/patternfly-org/home-page-dist/_site/index.html) page.

**Comment by mcarrano** [comment]
I like this. Would like to also have @LHinson take a look before we commit to this approach. The only issue I noticed with the demo is that it looks like the caret icon is not completely centered in the round button. See attached screenshot.
![screen shot 2018-05-24 at 9 03 42 am](https://user-images.githubusercontent.com/6619860/40487300-b5a63d00-5f31-11e8-98ef-a4f97170e1d2.png)

**Comment by dabeng** [comment]
Hi @mcarrano I have adjusted the position of caret icon.

**Comment by dabeng** [comment]
Hi @LHinson Could you review this approach before I sent PR

**Comment by mcarrano** [comment]
Looks good to me. @LHinson did you ever have a chance to look at this?

**Comment by LHinson** [comment]
I think this look great! No comments. Is there a PR associated that I should be approving?
